src/OnboardingSPA/pages/Steps/Ecommerce/StepProducts/index.js

```diff
@@ -17,84 +20,111 @@ const StepProducts = () => {
 		setDrawerActiveView,
 		setSidebarActiveView,
 		setCurrentOnboardingData,
-	} = useDispatch(nfdOnboardingStore);
+	} = useDispatch( nfdOnboardingStore );
 
-	let currentData = useSelect((select) =>
-		select(nfdOnboardingStore).getCurrentOnboardingData()
+	const currentData = useSelect( ( select ) =>
```

Comment:
was switching the keyword from let to const intentional or due to lint? Although, const seems more apt.

Reply:
Yeah, That happened cause I ran lint and that is the expected standard.